### PR TITLE
Implemented abstract methods must always be public.

### PR DESCRIPTION
**Pull request by CyrusNajmabadi**
Fixes https://github.com/dotnet/roslyn/issues/11444

### CODE REVIEW

**Comment by CyrusNajmabadi** [comment]
tagging @dotnet/roslyn-ide

File: src/EditorFeatures/VisualBasicTest/Diagnostics/ImplementInterface/ImplementInterfaceTests.vb [hunk 1]
```diff
@@ -2177,5 +2176,31 @@ NewLines("Imports System \n Imports System.Collections.Generic \n Class Program(
...
+End Interface
+
+Public MustInherit Class C
+    Implements IFace
+
+    Public MustOverride Sub IFace_M() Implements IFace.M
+    Public MustOverride Sub M()
+End Class",
+index:=1)
+        End Function
```
Comment (balajikris) [inline src/EditorFeatures/VisualBasicTest/Diagnostics/ImplementInterface/ImplementInterfaceTests.vb]: want to verify no errors exist after the fix is applied?

**Comment by balajikris** [comment]
:+1:

**Comment by Pilchie** [comment]
Technically it doesn't have to be public - it could be Protected or Friend, but I'm fine with this fix.

**Comment by Pilchie** [comment]
:+1:

**Comment by CyrusNajmabadi** [comment]
Good point.  But yes, public is a sensible default for implemented interface methods.

**Pull request merged**